spec/diddocuments.md

```diff
@@ -772,6 +870,7 @@ This section focuses on delegation relationships between KERI AIDs. [DID Documen
 #### Delegation key state events
 1. All delegation relationships MUST start with a delegated inception event.
 1. Any change to the [[ref: Delegated inception event]] key state or delegated rotation event key state MUST be the result of a delegated rotation event.
+1. See [Verification Relationships](#verification-relationships) for a detailed description of how KERI delegation is reflected in the DID document.
 
 > Delegated [[ref: inception event]]: Establishes a delegated identifier. Either the delegator or the delegate can end the delegation commitment.
```

Comment:
> Either the delegator or the delegate can end the delegation commitment.

This statement is not correct.

It is unclear to me how a delegate would end the "authorization" provided by delegation.
I'm not sure how a delegator would end delegation - potentially through credential issuance.

It raises questions that need clarification but I do not think this language belongs in the specification.